AzureMonitorAgent/ama_tst/modules/errors.py

```diff
@@ -45,8 +45,8 @@
     ERR_PKG_MANAGER : "This system does not have a supported package manager. Please install 'dpkg' or 'rpm' "\
           "and run this troubleshooter again.",
     ERR_MULTIPLE_AMA : "There is more than one instance of AMA installed, please remove the extra AMA packages.",
-    ERR_AMA_INSTALL : "AMA package isn't installed correctly.",
-    ERR_SUBMODULE_INSTALL : "Submodule(s) {0} not installed correctly.",
+    ERR_AMA_INSTALL : "AMA package isn't installed correctly.\n\nError Details: \n{0}",
```

Comment:
Seems like in not all cases will the error details will be filled; e.g. `check_pkgs.py` line 13

Reply:
I checked the exceptions that should not be passed and display those error messages in the latest version.